modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/checkbox/aui/page.jsp

```diff
@@ -22,4 +22,4 @@
 	<%@ include file="/checkbox/extended_label.jspf" %>
 </liferay-util:buffer>
 
-<aui:input checked="<%= checked %>" data-qa-id="<%= name %>" disabled="<%= disabled %>" id="<%= id %>" ignoreRequestValue="<%= true %>" label="<%= extendedLabel %>" name="<%= name %>" type="checkbox" />
\ No newline at end of file
+<aui:input checked="<%= checked %>" cssClass="label-item" data-qa-id="<%= name %>" disabled="<%= disabled %>" id="<%= id %>" ignoreRequestValue="<%= true %>" label="<%= extendedLabel %>" name="<%= name %>" type="checkbox" />
```

Comment:
Can we update this to use custom checkboxes? It will fix this issue and align it to Lexicon styles. This  should look like:

```jsp
<div class="form-group">
	<div class="custom-checkbox custom-control">
		<label>
			<aui:input checked="<%= checked %>" cssClass="custom-control-input" data-qa-id="<%= name %>" disabled="<%= disabled %>" id="<%= id %>" ignoreRequestValue="<%= true %>" label="" name="<%= name %>" type="checkbox" wrappedField="<%= true %>" />
			<%= extendedLabel %>
		</label>
	</div>
</div>
```

And `extended_label.jspf` should look like:

```jsp
<span class="custom-control-label">
	<span class="custom-control-label-text">
		<strong>
			<%= HtmlUtil.escape(label) %><%= separator %>
		</strong>

		<liferay-staging:popover
			id="<%= popoverName %>"
			text="<%= popoverText %>"
			title="<%= label %>"
		/>

		<c:if test="<%= items > 0 %>">
			<span class="staging-taglib-checkbox-items"><%= items + StringPool.SPACE + LanguageUtil.get(request, "items") %></span>
		</c:if>

		<c:if test="<%= deletions > 0 %>">
			<span class="deletions staging-taglib-checkbox-deletions"><%= deletions + StringPool.SPACE + LanguageUtil.get(request, "deletions") %></span>
		</c:if>

		<c:if test="<%= Validator.isNotNull(description) %>">
			<span class="staging-taglib-checkbox-description"><%= HtmlUtil.escape(description) %></span>
		</c:if>

		<c:if test="<%= Validator.isNotNull(warning) %>">
			<span class="staging-taglib-checkbox-warning"><%= HtmlUtil.escape(warning) %></span>
		</c:if>

		<c:if test="<%= Validator.isNotNull(suggestion) %>">
			<span class="staging-taglib-checkbox-suggestion"><%= HtmlUtil.escape(suggestion) %></span>
		</c:if>
	</span>
</span>
```

Reply:
A screenshot of what it looks like
![publish-to-live](https://user-images.githubusercontent.com/788266/191374002-1a746f7c-c887-4a83-bf0c-99713cf62be8.jpg)